common/txmgr/broadcaster.go

```diff
@@ -680,7 +693,9 @@ func (eb *Broadcaster[CHAIN_ID, HEAD, ADDR, TX_HASH, BLOCK_HASH, SEQ, FEE]) next
 	return etx, nil
 }
 
-func (eb *Broadcaster[CHAIN_ID, HEAD, ADDR, TX_HASH, BLOCK_HASH, SEQ, FEE]) tryAgainBumpingGas(ctx context.Context, lgr logger.Logger, txError error, etx txmgrtypes.Tx[CHAIN_ID, ADDR, TX_HASH, BLOCK_HASH, SEQ, FEE], attempt txmgrtypes.TxAttempt[CHAIN_ID, ADDR, TX_HASH, BLOCK_HASH, SEQ, FEE], initialBroadcastAt time.Time, retry int) (err error, retryable bool) {
+// replaceAttemptWithBumpedGas perform the replacement of the existing tx attempt with a new bumped fee attempt,
+// and it returns three values: new attempt, error, and retryable flag
```

Comment:
I think it's obvious to the user what gets returned, so it's better if we keep the description short and informative.
```suggestion
// replaceAttemptWithBumpedGas performs the replacement of the existing tx attempt with a new bumped fee attempt.
```